Add blog post: TANF modeling for all 50 states and DC (#695)
PolicyEngine/policyengine-app-v2: app/src/data/posts/articles/tanf-50-state-coverage.md

PolicyEngine now models Temporary Assistance for Needy Families (TANF) eligibility and benefit calculations for all 50 states and Washington, D.C. This completes our nationwide coverage of the federal block grant program that provides cash assistance to low-income families with children, with each state's specific rules encoded in our open-source rules engine.

## How we built it

Each state administers TANF under its own rules, setting benefit levels, income disregards, asset limits, and eligibility criteria independently. Encoding 51 distinct policy configurations required a systematic approach.

> We used the [multi-agent AI workflow](multi-agent-workflows-policy-research) we described in an earlier post. Claude Code agents handled the repetitive components of the implementation: reading state policy manuals, structuring parameters, drafting variable logic, and generating test cases. Human reviewers validated each state's rules against official documentation and resolved cases where policy language was ambiguous or where program interactions required judgment.

Reviewer: ```suggestion
We used the [multi-agent AI workflow](multi-agent-workflows-policy-research) we described in this article. Claude Code agents handled the repetitive components of the implementation: reading state policy manuals, structuring parameters, drafting variable logic, and generating test cases. Human reviewers validated each state's rules against official documentation and resolved cases where policy language was ambiguous or where program interactions required judgment.
```

> This workflow allowed us to maintain consistent code quality and documentation standards across all 51 implementations while reducing the time required for each additional state.

Reviewer: ```suggestion
This workflow allowed us to maintain consistent code quality and documentation standards across all 51 implementations while significantly reducing the time required for each additional state.
```

Reviewer: Only time we have been able to use the word significantly.
